README.md
netrunner-se-firefox-profile
============================

Firefox Profile for Netrunner SE


debian/changelog
netrunner-se-firefox-profile (1.0.1) raring; urgency=low

  * Removed all previous bookmarks in toolbar
  * Added bookmark to The hidden Wiki, TORCH and Encyclopedia Dramatica

 -- Leszek Lesner <leszek@zevenos.com>  Wed, 31 Jul 2013 12:37:58 +0200

netrunner-se-firefox-profile (1.0.0) raring; urgency=low

  * Based on the netrunner-artwork-firefox package
  * Added HTTPS Everywhere extension
  * Added FoxyProxy extension
  * Added NoScript extension
  * Set DuckDuckGo as default search engine

 -- Leszek Lesner <leszek@zevenos.com>  Mon, 22 Jul 2013 10:38:51 +0200

netrunner-artwork-firefox (1.0.20) enigma-1306; urgency=low

  * New AdblockPlus changes

 -- Clement Lefebvre <root@linuxmint.com>  Fri, 05 Jul 2013 10:58:44 +0200

netrunner-artwork-firefox (1.0.19) enigma-1306; urgency=low

  * RC2

 -- Frédéric Gaudet <fred@linuxmint.com>  Mon, 27 May 2013 11:14:14 +0200

netrunner-artwork-firefox (1.0.18) enigma-1306; urgency=low

  * RC1

 -- Frédéric Gaudet <fred@linuxmint.com>  Fri, 24 May 2013 14:06:27 +0200

netrunner-artwork-firefox (1.0.17) enigma-1306; urgency=low

  * BETA4

 -- Frédéric Gaudet <fred@linuxmint.com>  Thu, 16 May 2013 15:59:44 +0200

netrunner-artwork-firefox (1.0.16) enigma-1306; urgency=low

  * Update against FF20.0

 -- Frédéric Gaudet <fred@linuxmint.com>  Thu, 11 Apr 2013 14:56:22 +0200

netrunner-artwork-firefox (1.0.15) enigma-1306; urgency=low

  * Initial enigma release

 -- Frédéric Gaudet <fred@linuxmint.com>  Fri, 15 Mar 2013 10:24:54 +0100

netrunner-artwork-firefox (1.0.14) dryland-te; urgency=low

  * Update against version 18.0

 -- Frédéric Gaudet <fred@linuxmint.com>  Tue, 15 Jan 2013 14:54:38 +0100

netrunner-artwork-firefox (1.0.13) dryland-te; urgency=low

  * Update against version 17.0.1 

 -- Frédéric Gaudet <fred@linuxmint.com>  Wed, 19 Dec 2012 13:11:09 +0100

netrunner-artwork-firefox (1.0.12) dryland-te; urgency=low

  * Webapps not maximized

 -- Frédéric Gaudet <fred@linuxmint.com>  Mon, 10 Dec 2012 21:52:08 +0100

netrunner-artwork-firefox (1.0.11) dryland-te; urgency=low

  * Initial Dryland TE release

 -- Frédéric Gaudet <fred@linuxmint.com>  Tue, 23 Oct 2012 11:22:12 +0200

netrunner-artwork-firefox (1.0.10) dryland-se; urgency=low

  * Removed Tabmix Plus
  * Removed Easylist from AdBlockPlus
  * Upgraded AdBlockPlus
  * Removed Oxygen KDE Options

 -- Clement Lefebvre <root@linuxmint.com>  Tue, 21 Aug 2012 11:47:00 +0100

netrunner-artwork-firefox (1.0.9) dryland-se; urgency=low

  * Manage torrent file association
  * Change initial homepage

 -- Frederic Gaudet <fred@linuxmint.com>  Thu, 19 Jul 2012 10:12:31 +0200

netrunner-artwork-firefox (1.0.8) dryland-se; urgency=low

  * Initial dryland-se release

 -- Fred <fred@linuxmint.com>  Fri, 13 Apr 2012 10:10:28 +0200

netrunner-artwork-firefox (1.0.7) dryland; urgency=low

  * Misc fixes

 -- Phoenix <phoenix@netrunner-os.com>  Wed,  22 Dec 2011 16:16:00 +0000

netrunner-artwork-firefox (1.0.6) dryland; urgency=low

  * Misc fixes

 -- Phoenix <phoenix@netrunner-os.com>  Wed,  22 Dec 2011 16:16:00 +0000

netrunner-artwork-firefox (1.0.5) dryland; urgency=low

  * Misc fixes

 -- Phoenix <phoenix@netrunner-os.com>  Wed,  11 Dec 2011 18:16:00 +0000

netrunner-artwork-firefox (1.0.4) dryland; urgency=low

  * Misc fixes

 -- Phoenix <phoenix@netrunner-os.com>  Wed,  11 Dec 2011 18:16:00 +0000

netrunner-artwork-firefox (1.0.3) dryland; urgency=low

  * Fix toolbar

 -- Phoenix <phoenix@netrunner-os.com>  Wed,  5 Dec 2011 10:48:00 +0000

netrunner-artwork-firefox (1.0.2) dryland; urgency=low

  * Fix extensions

 -- Phoenix <phoenix@netrunner-os.com>  Wed,  2 Dec 2011 16:48:00 +0000

netrunner-artwork-firefox (1.0.1) dryland; urgency=low

  * Add extensions

 -- Phoenix <phoenix@netrunner-os.com>  Wed,  2 Dec 2011 16:48:00 +0000

netrunner-artwork-firefox (1.0.0) dryland; urgency=low

  * Initial release

 -- Phoenix <phoenix@netrunner-os.com>  Wed,  30 Nov 2011 13:48:00 +0000



etc/skel/.mozilla/firefox/se-default/extensions/{b9db16a4-6edc-47ec-a1f4-b86292ed211d}/defaults/preferences/prefs-dwhelper.js
/******************************************************************************
 *            Copyright (c) 2006-2011 Michel Gutierrez. All Rights Reserved.
 ******************************************************************************/

pref("dwhelper.download-mode", "controlled");
pref("dwhelper.download.controlled.max", 2);
pref("dwhelper.first-time", true);
pref("dwhelper.alert-update-version", "");
pref("dwhelper.menu-http-expiration",60000);
pref("dwhelper.icon-animation",true);
pref("dwhelper.ytlinks-enabled",true);
pref("dwhelper.medialink-extensions","jpg|jpeg|gif|png|mpg|mpeg|avi|rm|wmv|mov|flv|mp3");
pref("dwhelper.mediareq-extensions","flv|ram|mpg|mpeg|avi|rm|wmv|mov|asf|mp3|rar|movie|divx|rbs|mp4|mpeg4");
pref("dwhelper.adult",false);
pref("dwhelper.checked-adult",false);
pref("dwhelper.context-menu",true);
pref("extensions.{b9db16a4-6edc-47ec-a1f4-b86292ed211d}.description","chrome://dwhelper/locale/strings.properties");
pref("dwhelper.enable-plugins-method",false);
pref("dwhelper.disable-dwcount-cookie",false);
pref("dwhelper.mediaweight","");
pref("dwhelper.mediaweight-enabled",true);
pref("dwhelper.mediaweight-threshold",1048576);
pref("dwhelper.history-enabled",false);
pref("dwhelper.history-clearonexit",true);
pref("dwhelper.shorten-yt-address",true);
pref("dwhelper.exclude-ms-asf",true);
pref("dwhelper.cache-tracker",true);
pref("dwhelper.url-as-tooltip",true);
pref("dwhelper.force-cache",true);
pref("dwhelper.convert.keep-original",false);
pref("dwhelper.convert.keep-original-on-failure",true);
pref("dwhelper.quickkey","3;Q");
pref("dwhelper.quickkey.key","Q");
pref("dwhelper.quickkey.modifier","control shift");
pref("dwhelper.opendirkey","3;O");
pref("dwhelper.opendirkey.key","O");
pref("dwhelper.opendirkey.modifier","control shift");
pref("dwhelper.show-in-statusbar",false);
pref("dwhelper.trigger-min-size","100");
pref("dwhelper.convert-helper.notification",true);
pref("dwhelper.convert-helper.share-config",true);
pref("dwhelper.yt-check-hq",true);
pref("dwhelper.yt-prefer-hq",true);
pref("dwhelper.medialink-index-prefix",true);
pref("dwhelper.processor-keymap","0:download,2:convert-choice,3:quick-download");
pref("dwhelper.show-in-toolsmenu",true);
pref("dwhelper.quick-processor","quick-download");
pref("dwhelper.preferred-converter","ffmpeg");
pref("dwhelper.converter-path-ffmpeg","/usr/bin/ffmpeg");
pref("dwhelper.converter-path-avconv","/usr/bin/avconv");
pref("dwhelper.converter-path-mencoder","/usr/bin/mencoder");
pref("dwhelper.mp3tunes.username","");
pref("dwhelper.mp3tunes.password","");
pref("dwhelper.mp3tunes.phonenumber","");
pref("dwhelper.mp3tunes.enabled",false);
pref("dwhelper.highlight-media-from-current-page",true);
pref("dwhelper.mp3tunes.last-status","unchecked");
pref("dwhelper.system-menu-icons",true);
pref("dwhelper.donate-not-again",false);
pref("dwhelper.mp3tunes.show-no-converter-warning",true);
pref("dwhelper.mp3tunes.visible",false);
pref("dwhelper.yt-inpage",true);
pref("dwhelper.yt-inpage.default-processor","quick-download");
pref("dwhelper.mp3tunes.allow-video-upload",true);
pref("dwhelper.mp3tunes.upload.mobile","ask");
pref("dwhelper.mp3tunes.upload.locker","ask");
pref("dwhelper.ythq-formats","18,38,37,22,35,34,6,5,17,13,135,134,133,160,82,83,84,85,139,140,141,36,37,38");
pref("dwhelper.yt-log",false);
pref("dwhelper.yt-sig-perm-86","[[2,63],[82,83],[64,82],[63,64]]");
pref("dwhelper.yt-sig-perm-83","[[6,7],[3,6],[33,34],[7,24],[0,1],[25,33],[2,3],[34,53],[24,25],[54,83]]");
pref("dwhelper.yt-sig-perm-92","[[25,26],[3,25],[2,3],[26,42],[79,80],[43,79],[88,89],[80,83]]");
pref("dwhelper.media-host-blacklist","pop6.com|redlightcenter.com|dtiserv.com|mp3tunes.com|netflix.com");
pref("dwhelper.last-media-host-blacklist","pop6.com|redlightcenter.com|dtiserv.com|mp3tunes.com");
pref("dwhelper.share-blacklist",true);
pref("dwhelper.last-shared-blacklist","0");
pref("dwhelper.extended-download-menu",true);
pref("dwhelper.smartnamer.enabled",true);
pref("dwhelper.smartnamer.fname.keep-spaces",true);
pref("dwhelper.smartnamer.fname.keep-nonascii",true);
pref("dwhelper.smartnamer.fname.maxlen",64);
pref("dwhelper.smartnamer.manual-share",false);
pref("dwhelper.smartnamer.auto-share",true);
pref("dwhelper.twitter.enabled",true);
pref("dwhelper.twitter.username","");
pref("dwhelper.twitter.password","");
pref("dwhelper.twitter.tag-message",true);
pref("dwhelper.socialshare.enabled",true);
pref("dwhelper.medialink-max-hits",50);
pref("dwhelper.icon-click","sites");
pref("dwhelper.passwords-migrated",false);
pref("dwhelper.quickdownload-notify",true);
pref("dwhelper.seh-intro-proc.enable",true);
pref("dwhelper.smartnamer.max.name",128);
pref("dwhelper.smartnamer.max.descr",512);
pref("dwhelper.do-welcome",true);
pref("dwhelper.informer-spyware-notification",true);
pref("dwhelper.ythq-prefix",2);
pref("dwhelper.prefer-page-title-as-video-name",false);
pref("dwhelper.yt-fmt-switcher",false);
pref("dwhelper.social-share.enabled",true);
pref("dwhelper.social-share.tag-message",true);
pref("dwhelper.social-share.service.propeller.enabled",false);
pref("dwhelper.social-share.service.reddit.enabled",false);
pref("dwhelper.social-share.service.magnoliacom.enabled",false);
pref("dwhelper.social-share.service.newsvine.enabled",false);
pref("dwhelper.social-share.service.icerocket.enabled",false);
pref("dwhelper.social-share.service.misterwong.enabled",false);
pref("dwhelper.social-share.service.mixx.enabled",false);
pref("dwhelper.social-share.service.box.enabled",false);
pref("dwhelper.social-share.service.blinklist.enabled",false);
pref("dwhelper.social-share.service.identica.enabled",false);
pref("dwhelper.scap.load-timeout",200);
pref("dwhelper.scap.enabled",false);
pref("dwhelper.scap.start-pgm","");
pref("dwhelper.scap.start-args","");
pref("dwhelper.scap.stop-pgm","");
pref("dwhelper.scap.stop-args","");
pref("dwhelper.scap.file-ext","");
pref("dwhelper.scap.border-thickness",5);
pref("dwhelper.scap.ignore-cshelper",false);
pref("dwhelper.scap.stealth",true);
pref("dwhelper.scap.ignore-exit-value",true);
pref("dwhelper.startup-timer.widgets",1000);
pref("dwhelper.startup-timer.button",1000);
pref("dwhelper.yt-cookies.warning",true);
pref("dwhelper.yt-cookies2.warning",true);
pref("dwhelper.download-counter",true);
pref("dwhelper.file-extension-mode","ask");
pref("dwhelper.last-conv-license-status","unknown");
pref("dwhelper.show-expire-menu",false);
pref("dwhelper.yt-visitor-cookie","keep");
